Notre Dame Philosophical Reviews.js

```diff
@@ -0,0 +1,207 @@
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+var detectWeb, doWeb, scrape, testCases;
+
+detectWeb = function(doc, url) {
+  if (url.match(/^https:\/\/ndpr\.nd\.edu\/news\/[0-9]+/)) {
+    return 'journalArticle';
+  }
+  if (url.match(/^https:\/\/ndpr\.nd\.edu\/recent-reviews/)) {
+    return 'multiple';
+  }
+  if (url.match(/^https:\/\/ndpr\.nd\.edu\/search/)) {
+    return 'multiple';
+  }
+  return null;
+};
+
+scrape = function(doc, url) {
+  var abstractNote, author, bibliography, date, index, item, m, reviewedby, reviewer, title;
+  item = new Zotero.Item('journalArticle');
+  item.publication = 'Notre Dame Philosophical Reviews';
+  item.ISSN = '1538-1617';
+  item.url = doc.location.href;
+  date = doc.getElementsByClassName('pubdate')[0];
+  if (date) {
+    item.date = date.getAttribute('datetime') || '';
+    item.date = item.date.replace(/T.*/, '');
+    if (m = item.date.match(/^([0-9]{4})([0-9]{2})([0-9]{2})$/)) {
+      item.date = m[1] + "-" + m[2] + "-" + m[3];
+    }
+  }
+  item.creators = [];
+  title = doc.getElementsByClassName('entry-title')[0];
+  if (title) {
+    title = title.textContent;
+    item.title = "Review of <i>" + title + "</i>";
+
+    /* shorten to before first colon */
+    index = title.indexOf(':');
+    if (index >= 0) {
+      title = title.substr(0, index);
+    }
+
+    /* shorten to after first question mark */
+    index = title.indexOf('?');
+    if (index >= 0) {
+      index++;
+      if (index !== title.length) {
+        title = title.substr(0, index);
+      }
+    }
+    title = "Review of <i>" + title + "</i>";
+    if (title !== item.title) {
+      item.shortTitle = title;
+    }
+  }
+  author = doc.getElementsByClassName('author')[0];
+  if (author) {
+    item.creators.push(Zotero.Utilities.cleanAuthor(author.textContent, 'reviewedAuthor'));
+  }
+  reviewer = doc.getElementsByClassName('reviewers')[0];
+  if (reviewer) {
+    reviewedby = doc.getElementsByClassName('reviewed-by')[0];
+    if (reviewedby) {
+      reviewer.removeChild(reviewedby);
+    }
+    reviewer = reviewer.textContent.replace(/,.*/, '').trim();
+    item.creators.push(Zotero.Utilities.cleanAuthor(reviewer, 'author'));
+  }
+  abstractNote = doc.getElementsByClassName('entry-content')[0];
+  if (abstractNote) {
+    item.abstractNote = abstractNote.textContent;
+  }
+  bibliography = doc.getElementsByClassName('bibliography')[0];
+  bibliography = ((bibliography && bibliography.textContent) || '').trim();
+  if (bibliography) {
+    item.notes = [bibliography];
```

Comment:
notes are arrays of objects, not arrays of strings. Use
`item.notes.push({note:bibliography});`